### PR TITLE
invoke pip.compile

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -19,7 +19,7 @@ cffi==1.2.1               # via bcrypt
 click==5.1
 docutils==0.12            # via botocore, readme
 hiredis==0.2.0
-html5lib==0.999999        # via bleach
+html5lib==0.9999999       # via bleach
```
Comment (mattrobenolt) [inline requirements.txt]: lol

Comment (alex) [inline requirements.txt]: it's fine. probably no software in this stack represents versions as floats.

**Comment by ewdurbin** [comment]
Does this resolve the invalid IPv6 errors? Or is it just keeping up with the releases of html5lib?

**Comment by dstufft** [comment]
It should resolve them.

**Pull request merged**